cmd/launch.go

```diff
@@ -437,6 +468,9 @@ func ReadNetworkConfiguration(h *ec2helper.EC2Helper, simpleConfig *config.Simpl
 		if defaultsConfig.Vpc != nil {
 			defaultVpcId = *defaultsConfig.Vpc.VpcId
 		}
+		if defaultsConfig.SecurityGroups != nil {
+			defaultSecurityGroups = defaultsConfig.SecurityGroups
+		}
```

Comment:
Yes and no. I added in defaults for the security groups since the question could use the properly now. I made the ReadSubnetPlaceholder function return a boolean again to make it like the other functions since it had error checking now. 

This wont work if added into prod now because I changed how the ReadSecurityGroups method works

Reply:
All right, thanks for the explanation.